clang/lib/Format/Format.cpp

```diff
@@ -3872,6 +3887,10 @@ reformat(const FormatStyle &Style, StringRef Code,
     return IntegerLiteralSeparatorFixer().process(Env, Expanded);
   });
 
+  Passes.emplace_back([&](const Environment &Env) {
```

Comment:
Add a check, if any option is not `Leave` (I'd add a member function to the struct), so that the people who don't use this feature don't pay for it.